.github/workflows/html-preview.yml

```diff
@@ -1,23 +1,17 @@
 name: html-preview
-on:
-  push:
-    branches:
-      - main
-      - develop
-  pull_request:
-    branches:
-      - main
-      - develop
-  workflow_dispatch: # be able to trigger workflow manually
```

Comment:
Tbh, I do not know how to combine the `[status]` and `workflow_dispatch` so that it results in a valid workflow file. I could not find any resources on that and even brute forcing with this online checker https://rhysd.github.io/actionlint/ did not work.
But I also do not see any scenario, in which the workflow has to be dispatched manually. This is only for linking the current CI artifact. The CI will run completely independent of this workflow

Reply:
thanks for the answer.
Then lets merge it like that.